src/experimental/ks-installer/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+description: An open source software developed by kubesphere.To help install kubesphere on a kubernetes cluster
```

Comment:
An open source software developed by kubesphere.To help install kubesphere on a kubernetes cluster => The helm chart of KubeSphere, supports installing KubeSphere on existing Kubernetes.